### DISCUSSION TITLE
Automation POST Query

### DISCUSSION THREAD

**Discussion by organizedslop** [Budibase/budibase#7796] [Help] [answered]
I'm trying to set up an automation to send a POST query but keep getting "Please provide session data or access token". The query is working perfectly in the query editor, but in the automation tester and the published app I'm getting this response. Has anybody dealt with anything similar?

**Comment by melohagan** [comment, marked as answer]
Hey @organizedslop 

It sounds like your query relies on an SSO access token. In the query editor and in your app, this access token can be pulled from the currently logged in user.
The reason this is not possible in automations is because the automations run in the background and do not have and understanding of who is currently logged in.
